pkg/apis/policy/types.go

```diff
@@ -213,6 +213,10 @@ type PodSecurityPolicySpec struct {
 	// is allowed in the "Volumes" field.
 	// +optional
 	AllowedFlexVolumes []AllowedFlexVolume
+	// AllowedCSIDrivers is a whitelist of allowed CSI drivers.  This slice will be checked
+	// to verify which CSI driver is allowed to run in the cluster.
+	// +optional
+	AllowedCSIDrivers []AllowedCSIDriver
```

Comment:
is this a free form string or some pre-defined types in the api ?

Reply:
Driver name which is a string.